client/my-sites/checkout/src/hooks/use-create-payment-methods/index.tsx

```diff
@@ -5,8 +5,6 @@ import {
 	createGooglePayMethod,
 	createBancontactMethod,
 	createBancontactPaymentMethodStore,
-	createGiropayMethod,
-	createGiropayPaymentMethodStore,
```

Comment:
Where are these two methods defined? I can see that you are importing them here, but I don't see the originals removed in this diff. Am I missing it?

Reply:
They're defined in [a file I removed](https://github.com/Automattic/wp-calypso/pull/92099/files/5cd112d54d12d918aeeb304f0ada6c8ddfd997c7#diff-6f51d698b3145698b9fc5347d07467b929bc492845c2be924dfb0c18db21aca2L74-L91) `packages/wpcom-checkout/src/payment-methods/giropay.tsx`

Reply:
Thanks for clearing that up for me 😁